platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -181,7 +183,7 @@ function WorkList({ history, data: studies, isLoadingData, dataSource }) {
 
   const isFiltering = (filterValues, defaultFilterValues) => {
     return Object.keys(defaultFilterValues).some(name => {
-      return filterValues[name] !== defaultFilterValues[name];
+      return !isEqual(filterValues[name], defaultFilterValues[name]);
```

Comment:
We know our filter values. I think we can do strict equality comparison for everything except the date range object and the modalities array? For those, we can use the key value to indicated we need to compare differently?

Reply:
I just switch to using `!isEqual(filterValues, defaultFilterValues)` since I don't see any reason why this isn't easier.